File: MainServer/src/main/java/via/pro3/mainserver/DTOs/PrescriptionDto.java
package via.pro3.mainserver.DTOs;

public class PrescriptionDto {

    public int id;
    public String diagnosis;
    public String medication;
    public String recommendations;
    public String date;
    public String time;
    public String patientcpr;
    public String doctorid;

    public PrescriptionDto(int id, String diagnosis, String medication, String recommendations, String date, String time, String patientcpr, String doctorid) {
        this.id = id;
        this.diagnosis = diagnosis;
        this.medication = medication;
        this.recommendations = recommendations;
        this.date = date;
        this.time = time;
        this.patientcpr = patientcpr;
        this.doctorid = doctorid;
    }

    public int getId() {
        return id;
    }

    public String getDiagnosis() {
        return diagnosis;
    }

    public String getMedication() {
        return medication;
    }

    public String getRecommendations() {
        return recommendations;
    }

    public String getDate() {
        return date;
    }


    public String getTime() {
        return time;
    }

    public String getPatientCpr() {
        return patientcpr;
    }


    public String getDoctorId() {
        return doctorid;
    }
}

File: Patient/Patient/DTOs/PrescriptionDto.cs
namespace Patient.DTOs
{
    public class PrescriptionDto
    {
        public int id { get; set; }
        public string diagnosis { get; set; }
        public string medication { get; set; }
        public string recommendations { get; set; }
        public string date { get; set; }
        public string time { get; set; }
        public string patientcpr { get; set; }
        public string doctorid { get; set; }

        public PrescriptionDto(int id, string diagnosis, string medication, string recommendations, string date, string time, string patientcpr, string doctorid)
        {
            this.id = id;
            this.diagnosis = diagnosis;
            this.medication = medication;
            this.recommendations = recommendations;
            this.date = date;
            this.time = time;
            this.patientcpr = patientcpr;
            this.doctorid = doctorid;
        }
    }
}